test/dlc_tests/benchmark/sagemaker/tensorflow/training/resources/tf_sm_benchmark.py

```diff
@@ -0,0 +1,60 @@
+import argparse
```

Comment:
This script is a combination of https://github.com/akhilmehra/sagemakerbenchmarking/blob/tf2/src/startit_tf_gpu.py and https://github.com/akhilmehra/sagemakerbenchmarking/blob/tf2/src/startit_tf_cpu.py